app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+import requests
+
```

Comment:
Consider adding a docstring at the beginning of the `get_weather` function to describe its purpose and usage. This will improve code readability and maintainability.